src/sumo/wrapper/sumo_client.py

```diff
@@ -398,6 +403,35 @@ def getLogger(self, name):
             pass
         return logger
 
+    def create_shared_access_key_for_case(self, case_uuid):
+        """Creates and stores a shared access key that can be used to access
+        the case identified by *case_uuid*, in the current Sumo environment.
+
+        This shared access key can then be used by instantiating
+        SumoClient with the parameter case_uuid set accordingly.
+
+        Args:
+            case_uuid: the uuid for a case.
+
+        Side effects:
+            Creates a new file in ~/.sumo, named {app_id}+{case_uuid}
+        """
+        token = self.get(
+            f"/objects('{case_uuid}')/make-shared-access-key"
+        ).text
+        self.auth.store_shared_access_key_for_case(case_uuid, token)
+
+    def client_for_case(self, case_uuid):
+        """Instantiate and return new SumoClient for accessing the
+        case identified by "case_uuid*."""
```

Comment:
Typo? I guess `*case_uuid*` was the intention.

Reply:
Absolutely... fixed.